step.js

```diff
@@ -104,17 +107,33 @@ function renderRocket() {
 }
   
 function renderShip() {
-    airfighter.element.style.left = airfighter.x;
-    airfighter.element.style.top = airfighter.y;
-    airfighter.health.element.style.left = airfighter.x;
-    airfighter.health.element.style.top = airfighter.y;
-    airfighter.health.element.style.width = airfighter.width;
-    airfighter.health.element.style.height = airfighter.height*0.1;
-    airfighter.healthtext.element.value = airfighter.health.element.value / airfighter.health.element.max;
-    airfighter.healthtext.element.innerHTML = `${airfighter.health.element.value} / ${airfighter.health.element.max} HP`;
-    airfighter.healthtext.element.style.left = airfighter.x;
-    airfighter.healthtext.element.style.top = airfighter.y - 35;
-    airfighter.healthtext.element.style.width = airfighter.width;
+  airfighter.element.style.left = airfighter.x;
+  airfighter.element.style.top = airfighter.y;
+  airfighter.health.element.style.left = airfighter.x;
+  airfighter.health.element.style.top = airfighter.y;
+  airfighter.health.element.style.width = airfighter.width;
+  airfighter.health.element.style.height = airfighter.height*0.1;
+  airfighter.healthtext.element.value = airfighter.health.element.value / airfighter.health.element.max;
+  airfighter.healthtext.element.innerHTML = `${airfighter.health.element.value} / ${airfighter.health.element.max} HP`;
+  airfighter.healthtext.element.style.left = airfighter.x;
+  airfighter.healthtext.element.style.top = airfighter.y - 35;
+  airfighter.healthtext.element.style.width = airfighter.width;
+
+  if (airfighter.vx > 0 && airfighter.isShipMovingRight == false) {
```

Comment:
Please create a variable for this condition, like this:
```
const something = (airfighter.vx > 0 && airfighter.isShipMovingRight == false);
if (something) {
  ...
}
```
This way it will be easier to understand the code.